docs/features/operators.md

```diff
@@ -0,0 +1,206 @@
+# Operators
+
+Operators are very useful for manipulating values, this will go though all of the operators currently in N:
+
+### `OR`
+The OR operator (`||` or `|`) can be used on two booleans or two ints, it will either output a bool or a int depending on its inputs.
+
+When the inputs are booleans it will return true if one of the values are `true`, the `||` variation is the standard for bools:
+```js
+let val = true || false // true
+
+let val1 = false || false // false
+```
+
+When the inputs are ints it will return an int that has all of the bits combined, the `|` variation is the standard for ints:
+```js
+let val1 = 0b1100 // 12
+let val2 = 0b0111 // 7
+
+let val = val1 | val2 // 15 or:
+/*
+  1100
+| 0111 =
+  1111
+*/
+```
+
+### `AND`
+The AND operator (`&&` or `&`) can be used on two booleans or two ints, it will either output a bool or a int depending on its inputs.
```

Comment:
```suggestion
The AND operator (`&&` or `&`) can be used on two booleans or two ints, it will either output a bool or an int depending on its inputs.
```